src/styles/variables.scss

```
@@ -0,0 +1,14 @@
+$nav-active-color: #00acdc;
+$header-color: #fff;
+$header-font-size: 12px;
+$nav-text-color: #060b35;
+$card-height: 408px;
+$card-width: 200px;
+$button-width: 166px;
+$button-height: 40px;
+$border-color: #f3f3f3;
+$main-text-color: #060b35;
+$secondary-text-color: #616070;
+$button-blue-color: #00acdc;
+$card-color: #fff;
```

Don't need to duplicate variables with the same values
```suggestion
$secondary-text-color: #616070;
```